lib/hmmlearn/base.py

```diff
@@ -641,8 +747,82 @@ def _initialize_sufficient_statistics(self):
                  'trans': np.zeros((self.n_components, self.n_components))}
         return stats
 
-    def _accumulate_sufficient_statistics(self, stats, X, framelogprob,
+    def _accumulate_sufficient_statistics(self, stats, X, lattice,
                                           posteriors, fwdlattice, bwdlattice):
+        """Updates sufficient statistics from a given sample.
+
+        Parameters
+        ----------
+        stats : dict
+            Sufficient statistics as returned by
+            :meth:`~base._BaseHMM._initialize_sufficient_statistics`.
+
+        X : array, shape (n_samples, n_features)
+            Sample sequence.
+
+        lattice : array, shape (n_samples, n_components)
+            Probabilities OR Log Probabilities of each sample
+            under each of the model states.  Depends on the choice
+            of implementation of the Forward-Backward algorithm
+
+        posteriors : array, shape (n_samples, n_components)
+            Posterior probabilities of each sample being generated by each
+            of the model states.
+
+        fwdlattice, bwdlattice : array, shape (n_samples, n_components)
+            Log-forward and log-backward probabilities.
+        """
+
+        impl = {
+            "scaling": self._accumulate_sufficient_statistics_scaling,
+            "log": self._accumulate_sufficient_statistics_log,
+        }[self.implementation]
+
+        return impl(stats=stats, X=X, lattice=lattice, posteriors=posteriors,
+                    fwdlattice=fwdlattice, bwdlattice=bwdlattice)
+
+    def _accumulate_sufficient_statistics_scaling(self, stats, X, lattice,
+                                                  posteriors, fwdlattice, bwdlattice):
+        """Updates sufficient statistics from a given sample.
```

Comment:
I would just say "Implementation of \`_accumulate_sufficient_statistics\` for \`\`implementation = "scaling"\`\`." and not bother with reproducing the docstring (which can otherwise easily go out of sync).  ditto for log.